src/main/groovy/org/robolectric/gradle/Configuration.groovy

```diff
@@ -0,0 +1,67 @@
+package org.robolectric.gradle
+
+import org.gradle.api.Project
+import com.android.build.gradle.AppPlugin
+import com.android.build.gradle.LibraryPlugin
+import com.android.build.gradle.api.BaseVariant
+
+class Configuration {
+    private final Project project
+    private final boolean hasAppPlugin
+    private final boolean hasLibPlugin
+    private static final String DEBUG_VARIANT_NAME = "debug"
```

Comment:
To confirm:  the tests are not able to find my resources/assets/manifest with this hard-coded "debug" variant name.  When I change the constant to "fullDebug" as a test, it does work as expected.

However, now the Google issue I linked in #128 is preventing my test resources (Config.properties, and some mock .json files for some of the API-parsing tests) from being found at runtime: https://code.google.com/p/android/issues/detail?id=136013.  But that's a Google issue, not something that Robolectric should fix.